nifi-frontend/src/main/frontend/apps/nifi/src/app/state/shared/index.ts

```diff
@@ -132,6 +132,12 @@ export interface EditControllerServiceDialogRequest {
     history?: ComponentHistory;
 }
 
+export interface MoveControllerServiceDialogRequest {
+    id: string;
+    controllerService: ControllerServiceEntity;
+    processGroupFlow: any;
+}
```

Comment:
Like the new `Move` component, I think this dialog request should be moved into the `flow-designer` page. It's not applicable outside of that page.

Reply:
`processGroupFlow` shouldn't need to be `any` for this dialog request. Admittedly there is some instances of `any` based on the underlying data model the back end uses but in this case there already is a defined type for this field that should be referenced.

Reply:
Moved this to flow-designer. Also, changed to use the ProcessGroupFlow model.